src/oc/web/stores/comment.cljs

```diff
@@ -245,6 +244,7 @@
             sorted-comments-data (sort-comments new-comments-data)
             org-slug (name (nth entry-key 0))
             board-slug (name (nth entry-key 2))
+            board-key (dispatcher/board-data-key org-slug board-slug)
```

Comment:
@thepug `board-data-key` is not the same as `current-board-key` because the latter distinguish between board and AP based on the router setup.
I think you should pass the board-key as a parameter and get it with the `current-board-key`.
WYT?

Reply:
we should probably update both keys (:all-posts , plus the board)

The idea here is that we eventually get rid of this data duplication.

Reply:
yeah but until we clean up the data duplication we need to at least update the correct data.. for this PR we should stick to the old logic: update the currently visible data path.